lib/eunit/test/tutf8.erl

```diff
@@ -0,0 +1,12 @@
+%% coding: utf-8
+
+-module(tutf8).
+
+-include_lib("eunit/include/eunit.hrl").
+
+foo_test_() ->
+	[
+ 	  {<<"_göod">>, fun() -> io:format("goöd"), io:format([128,64]), ?assert("gö"=="gö") end}
```

Comment:
No. `<<"_göod">>` is the converted representation of codepoints into a flat binary if the original encoding is unicode. In this case it fits latin1 (ö is codepoint 169, which requires no conversion for latin1), but take a character outside of the latin1 plane: 

```
1> <<"_göod💩">>.
<<"_göod©">>
```

This is because 💩 is the codepoint 128169 and © is 169. 128169 rem 256 = 169.
When the codepoint 128169 is entered as a binary, it is forced as a byte (rem 256) and yields a different character. 

And this:
```
2> <<"_göod💩"/utf8>>.
<<95,103,195,182,111,100,240,159,146,169>>
```
shows the proper utf8 encoding taking place.